downstream/modules/platform/proc-import-mesh-ca.adoc

```diff
@@ -0,0 +1,31 @@
+[id="proc-import-mesh-ca_{context}"]
+
+= Importing a Certificate Authority (CA) certificate
+
+A Certificate Authority (CA) is used to certify and sign execution nodes in an {AutomationMesh} environment. Users who wish to provide their own CA can do so by specifying the path to the certificate and the private RSA key file in the `inventory` file of their {PlatformName} installer.
```

Comment:
`is used to certify and sign execution nodes` might not be the right terminology. 

maybe `is used to verify and sign individual node certificates in an {AutomationMesh} environment`